src/auth/ntlm/SSPI/ntlm_sspi_auth.cc

```diff
@@ -164,178 +164,68 @@ Valid_Group(char *UserName, char *Group)
     return result;
 }
 
-char * AllocStrFromLSAStr(LSA_UNICODE_STRING LsaStr)
```

Comment:
Please separate these dead-code removal to a different PR than the enum refactor.

Reply:
Done. PR #1627